script.bash

```diff
@@ -129,6 +137,7 @@ delete() {
     echo -e "${GREEN}Hitting daemon-reload...${DEFAULT}"
     sudo systemctl daemon-reload
     echo -e "${GREEN}Service is deleted.${DEFAULT}"
+    # TODO: remove other data and requirements
```

Comment:
Remove this line, we don't need to remove the data (it will be controlled from `rubix-service` and normally we don't want to delete data), and requirements will acts as an individual app so that will be different one.